install.sh

```diff
@@ -358,10 +358,10 @@ download_airgap_tarball() {
     if [ -z "${INSTALL_RKE2_COMMIT}" ]; then
         return
     fi
-    AIRGAP_TARBALL_URL=${STORAGE_URL}/rke2-images.${SUFFIX}-${INSTALL_RKE2_COMMIT}.tar.zst
+    AIRGAP_TARBALL_URL=${STORAGE_URL}/rke2.${SUFFIX}-${INSTALL_RKE2_COMMIT}.tar.zst
     # try for zst first; if that fails use gz for older release branches
```

Comment:
Instead of changing this, another try for zst with the new name. With this change there will be three different variations that we might need to find depending on what release branch this commit was built from.

Reply:
ok! For both tarball and sha256sum, right?

Reply:
correct!

Reply:
The suffix I see in https://storage.googleapis.com is tar.gz though. Not tar.zst. I'll check rke2 without `-images` for that one